sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -0,0 +1,291 @@
+                    Assert.AreEqual(relation.Source.Category, "Dosage");
+                    Assert.AreEqual(relation.Source.ConfidenceScore, 1);
+                    Assert.AreEqual(relation.Source.Length, 5);
+                    Assert.AreEqual(relation.Source.Offset, 18);
+
+
+                    Assert.AreEqual(relation.Target.Text, "ibuprofen");
+                    Assert.AreEqual(relation.Target.Category, "MedicationName");
+                    Assert.AreEqual(relation.Target.ConfidenceScore, 1);
+                    Assert.AreEqual(relation.Target.Length, 9);
+                    Assert.AreEqual(relation.Target.Offset, 27);
+                }
+            }
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesWithLanguageTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = singleEnglish;
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(new List<string>() { document }, "en");
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            foreach (DocumentHealthcareResult result in resultCollection)
+            {
+                Assert.AreEqual(3, result.Entities.Count);
+                Assert.IsNotNull(result.Id);
+            }
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesWithTopParameter()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                Top = 1
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(batchDocuments, options);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(1, resultCollection.Count);
+            Assert.AreEqual(3, resultCollection[0].Entities.Count);
+            Assert.IsNotNull(resultCollection[0].Id);
+            Assert.AreEqual("1", resultCollection[0].Id);
+            Assert.AreEqual("100mg", resultCollection[0].Entities.FirstOrDefault().Text);
+            Assert.AreEqual("Dosage", resultCollection[0].Entities.FirstOrDefault().Category);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesWithSkipParameter()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                Skip = 1
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(batchDocuments, options);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.IsNotNull(resultCollection[0].Warnings);
```

Comment:
could you also that a validation for `Assert.IsNotNull(resultCollection[0].Errors);`